src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -437,6 +438,22 @@ def get_token(self):
 
         return self._token_provider.get_token()
 
+    @classmethod
+    def from_env(cls, name="default_env_connection"):
+        """Build connection from environment variables."""
```

Comment:
we should put clear docstring on the env key here, sdk user can avoid checking our docsite

Reply:
Got it.